src/constants.js

```diff
@@ -32,3 +32,15 @@ export const JOB_ASSET_NAMES = [
     'logcat.log',
     'video.mp4'
 ]
+
+export const DEFAULT_OPTIONS = {
+    user: process.env.SAUCE_USERNAME,
+    key: process.env.SAUCE_ACCESS_KEY,
+    headless: false,
+    region: 'us'
+}
+
+export const REGION_MAPPING = {
+    'us': '', // default endpoint
+    'eu': 'eu-central-1.'
```

Comment:
The new headless endpoint will be `https://us-east1.headless.saucelabs.com`. What should we now do, update the region mapping and refactor the `getSauceEndpoint` to something like

```js
export function getSauceEndpoint (region, headless) {
    const dcRegion = REGION_MAPPING[region] ? region : 'us'
    const prefix = headless ? `us-east1.headless` : REGION_MAPPING[dcRegion];
    return `${prefix}saucelabs.com`
}
```

Reply:
yeah, I will do something like this .. I guess we currently only support this one headless region

Reply:
Yes, there could be a possibility when headless gets more demands that we support multiple regions